pkg/aqua/command.go

```diff
@@ -68,61 +66,46 @@ func (c *command) Exec(imageRef ImageRef) (report ScanReport, err error) {
 		image = imageRef.WithTag()
 	}
 
-	flags := []string{
-		"--user", c.cfg.Username,
-		"--password", c.cfg.Password,
-		"--host", c.cfg.Host,
-		"--registry", c.cfg.Registry,
+	args := []string{
+		"scan",
+		"--checkonly",
 		"--dockerless",
-		"--jsonfile", reportFile.Name(),
-	}
-
-	if c.cfg.ScannerCLINoVerify {
-		flags = append(flags, "--no-verify")
-	}
-
-	if c.cfg.ScannerCLIShowNegligible {
-		flags = append(flags, "--show-negligible")
+		fmt.Sprintf("--user=%s", c.cfg.Username),
+		fmt.Sprintf("--password=%s", c.cfg.Password),
+		fmt.Sprintf("--host=%s", c.cfg.Host),
+		fmt.Sprintf("--registry=%s", c.cfg.Registry),
+		fmt.Sprintf("--no-verify=%t", c.cfg.ScannerCLINoVerify),
+		fmt.Sprintf("--show-negligible=%t", c.cfg.ScannerCLIShowNegligible),
+		fmt.Sprintf("--show-will-not-fix=%t", c.cfg.ScannerCLIShowWillNotFix),
+		fmt.Sprintf("--hide-base=%t", c.cfg.ScannerCLIHideBase),
+		fmt.Sprintf("--jsonfile=%s", reportFile.Name()),
+		image,
 	}
 
-	args := append([]string{"scan"}, flags...)
-	args = append(args, image)
-
-	log.WithFields(log.Fields{"exec": executable, "args": args}).Trace("Running scannercli")
+	// TODO Sanitize the args so the password is not printed no matter what's the log level!
+	log.WithFields(log.Fields{"exec": executable, "args": args}).Debug("Running scannercli")
 
 	cmd := exec.Command(executable, args...)
 
-	stderrBuffer := bytes.Buffer{}
-
-	cmd.Stderr = &stderrBuffer
-
-	stdout, err := cmd.Output()
-	if err != nil && cmd.ProcessState.ExitCode() != assurancePolicyCheckFailed {
```

Comment:
We don't need that anymore because of the `--checkonly` flagged passed to the `scannercli` command